provisioner/powershell/provisioner.go

```diff
@@ -423,8 +443,9 @@ func (p *Provisioner) createCommandTextNonPrivileged() (command string, err erro
 	}
 
 	p.config.ctx.Data = &ExecuteCommandTemplate{
-		Path: p.config.RemotePath,
-		Vars: envVarPath,
+		Path:          p.config.RemotePath,
+		Vars:          envVarPath,
+		WinRMPassword: p.getWinRMPassword(),
```

Comment:
this seems convenient, here, but with the above suggestion, the user could always use the `env` template function